libs/sdk-core/src/binding.rs

```diff
@@ -207,10 +207,11 @@ pub fn list_payments(
     filter: PaymentTypeFilter,
     from_timestamp: Option<i64>,
     to_timestamp: Option<i64>,
+    amount_of_payments: Option<u32>,
```

Comment:
This file should be generated using `make flutter_rust_bridge` in sdk-flutter directory